controller/misc/manager.py

```diff
@@ -47,6 +47,9 @@ def get_version_overview() -> List[ServiceVersionResult]:
 def has_updates() -> List[ServiceVersionResult]:
     return __updater_has_updates()
 
+def update_versions_to_newest() -> None:
```

Comment:
a comment to ensure that everyone know that is only sets the values in the database not the actual update logic (like in the updater service but here as well :)